tests/robot-cases/Group3-Upgrade/data.json

```diff
@@ -92,6 +92,7 @@
         "projects":[
            {
               "name":"project1",
+              "hasimage":"true",
```

Comment:
it could be set to bool, refer the has_admin

Reply:
got it.